vignettes/Model.Rmd

```diff
@@ -266,28 +312,42 @@ The remaining vignettes describe how to adjust sets of parameters through a numb
 
     -   `peak_season_offset()`: correlating timed interventions with seasonal malaria
 
-5.  [Vaccines](https://mrc-ide.github.io/malariasimulation/articles/Vaccines.html)
+4.  [Vaccines](https://mrc-ide.github.io/malariasimulation/articles/Vaccines.html)
 
     -   `set_mass_pev()`: implementation of a pre-erythrocytic vaccination intervention
-    -   `set_pev_epi()`: Implementation of a pre-erythrocytic vaccination intervention from a certain age
+    
+    -   `set_pev_epi()`: implementation of a pre-erythrocytic vaccination intervention from a certain age
+    
     -   `set_tbv()`: implementation of a transmission blocking vaccination intervention
 
-6.  [Vector Control: Bednets](https://mrc-ide.github.io/malariasimulation/articles/VectorControl.html)
+5.  [Vector Control: Bednets](https://mrc-ide.github.io/malariasimulation/articles/VectorControl_Bednets.html)
 
     -   `set_bednets()`: implementation of bednet distribution intervention
 
-7.  [Vector Control: Indoor Residual Spraying](https://mrc-ide.github.io/malariasimulation/articles/VectorControl.html)
+6.  [Vector Control: Indoor Residual Spraying](https://mrc-ide.github.io/malariasimulation/articles/VectorControl_IRS.html)
 
     -   `set_spraying()`: implementation of an indoor residual spraying intervention
 
-8.  [Matching PfPR2-10 to EIR](https://mrc-ide.github.io/malariasimulation/articles/EIRprevmatch.html)
+7.  [Mosquito Species](https://mrc-ide.github.io/malariasimulation/articles/SetSpecies.html)
+
+    -   `set_species()`: setting mosquito distribution
+
+8.  [Carrying Capacity](https://mrc-ide.github.io/malariasimulation/articles/Carrying-capacity.html)
```

Comment:
For some reason I can't find this vignette in this PR, perhaps it was renamed or deleted?